src/collections/integrations/docker.mdx

```diff
@@ -0,0 +1,75 @@
+faqAns: ["Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.","Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes.","Yes, as the service management plane, Meshery, provisions and manage the lifecycle of service meshes."]
+---
+import { IntegrationPageWrapper } from "../../sections/Meshery/Meshery-integrations/Integration.style";
+import docker from "../../assets/images/socialIcons/docker.svg";
+import arrow from "../../assets/images/app/forward-arrow.svg";
+import tick from "../../assets/images/app/tick.svg";
+import sliderImg_1 from "../../sections/Meshery/images/service mesh performance example.gif";
+import sliderImg_2 from "../../sections/Meshery/images/meshery_benchmark_screen.png";
+import sliderImg_3 from "../../sections/Meshery/images/meshery-configuration-management.png";
+import HowItWork from "../../sections/Meshery/Meshery-integrations/howItWork";
+
+import Button from "../../reusecore/Button";
+
+
+<IntegrationPageWrapper>
+    <section className="herosection">
+        <img src={docker} alt={_frontmatter.title} height={90} />
+        <h2>
+            {_frontmatter.title} With {" "}
+            <span
+                style={{
+                    color: "#00B39F",
+                }}
+            >
+          Meshery
+          </span>
+        </h2>
+        <p>{_frontmatter.subtitle}</p>
+    </section>
+    <section className="overview">
+        <div className="feature">
+            <h2>Features</h2>
+            {_frontmatter.featureList.map((feature, id) => {
+                return (
+                    <div key={id}>
+                        <img src={tick} alt="checkbox" />
+                        <p>{feature}</p>
+                    </div>
+                );
+            })}
+        </div>
+        <div className="content">
+            <h2>Overview</h2>
+            <p>
+                Connect {_frontmatter.title} with any of your favorite apps in just a few
+                clicks. Design, build, and automate anything for your work by
+                integrating apps like {_frontmatter.title} to create visual automatedcl
+                workflows. Choose from hundreds of ready-made design patterns or use
+                our no-code designer, MeshMap to connect to apps not yet in our
+                library.
+            </p>
+            <span>
+             <Button
+                 primary
+                 className="get-started"
+                 type="button"
+                 title="Get Started"
+                 url="#"
+             />
+            <span>
+              <a href={_frontmatter.docURL}>See Documentation</a>
+              <img src={arrow} alt="arrow" height={12} />
+            </span>
+          </span>
+        </div>
+    </section>
+    <HowItWork name={_frontmatter.title} sliderImg_1={sliderImg_1} sliderImg_2={sliderImg_2} sliderImg_3={sliderImg_3} />
+</IntegrationPageWrapper>
```

Comment:
This will become duplicate work when other integrations are added. Use graphql in the template file to fetch the integrations data and pass it to the Individual-Integrations section which should have this code.

In this MDX file, the body should only be the description that's it!
i.e,
```
Connect docker with any of your favorite apps in just a few
clicks. Design, build, and automate anything for your work by
integrating apps like {_frontmatter.title} to create visual automatedcl
workflows. Choose from hundreds of ready-made design patterns or use
our no-code designer, MeshMap to connect to apps not yet in our
library.
```

Reply:
Can you give me some more examples or anything that can help me figure out what exactly you want cause I am already fetching most of the data through GraphQL. I am putting only text in the body section except for one HowItWork component.  

If you want to prevent duplicate work, then I can make the overview section component, and I will pass that data into that component in the integration template through GraphQL. But if I do that, I will have nothing to put on the MDX body.  

So what exactly do you want me to do?

Reply:
I already mentioned what should be the content in the body.
Though, let me jump in now. Can't make it more clearer in words how it should be done even after mentioning so many times.

Reply:
I will push an update to this PR by tonight